defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+# defaults file for cockpit
+cockpit_version: cockpit-ovirt-dashboard-0.10.10-0.el7ev.noarch
```

Comment:
This is an incredibly specific version. Do we really want to default pin it this tightly, or should we just go with "cockpit-ovirt-dashboard" for the default and let the user pin it more specifically if they need to?